### ISSUE TITLE
Chrome devtools no longer appear in branch 3.15 for me

### ISSUE THREAD

**Issue by chrisruk** [LLK/scratch-desktop#151]
### Expected Behavior

Previously in scratch-desktop, when I ran:

```
npm start
```

Chrome devtools would also appear (when using branch 3.12).

### Actual Behavior

Chrome devtools no longer appear when the Scratch window opens.

### Steps to Reproduce

```
npm start
```

### Operating System and Browser

Linux

**Comment by apple502j** [comment]
Will it appear when you press Ctrl-Shift-I?

**Comment by chrisruk** [comment]
Yes! It does :) Thanks again for your help.

One last question, previously when making changes to code, I would do:

```
npm run watch-gui
```

And then:

```
npm start
```

I assume the watch-gui is no longer necessary as it seems to have been removed recently?

**Comment by cwillisf** [comment]
Both of these changes are intentional... sorry for the confusion!
* In #134 I made it so that the dev tools no longer open by default in any build, but they can be opened with the standard key combination (ctrl+shift+i on Windows or Linux, cmd+shift+i on macOS) even in release builds.
* In #138 (refined with #148) I made it so that you no longer need to pre-build scratch-gui -- it now builds as part of the regular webpack step for scratch-desktop. The `build-gui` and `watch-gui` scripts have been removed as they're no longer necessary.
